.ci/scripts/unit-test.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+source ./scripts/make/common.bash
+
+jenkins_setup
+
+make update
+mage goTestUnit
+
+# OUT_FILE="build/TEST-go-unit"
+# if [ -n "${TEST_COVERAGE}" ] && [ -f "${OUT_FILE}.cov" ]; then
+#   go install -modfile=tools/go.mod github.com/t-yuki/gocover-cobertura
+#   gocover-cobertura < "${OUT_FILE}.cov" > "${OUT_FILE}_cov.xml"
+# fi
```

Comment:
consider removing this part

Reply:
Regarding unit-tests, i prefer to keep them disabled and not removing them. Since we might need to run our prs on jenkins as well.

Reply:
It's an opinionated approach to this. I think you need to differentiate between application code & build code when it comes to quality standards.

This commented-out code will be needed in the near future & it is tested and working.  The feedback loop on this bit is quite tedious. Since this script can only run on Jenkins due to vault permissions.